packages/opentelemetry-context-zone-peer-dep/src/ZoneContextManager.ts

```diff
@@ -141,7 +141,12 @@ export class ZoneContextManager implements ContextManager {
   ) {
     const contextManager = this;
 
-    return function(this: {}, event: string, listener: Func<void>, opts?: any) {
+    return function (
```

Comment:
Did line length requirement change?

Reply:
need to check the gts plugin...

Reply:
In both versions (gts@2.0.0 && gts@1.1.2), no max-length were enforced. Default is `80`. Which is correct here because before it was `82`. I think the eslint parser is just better...